.github/workflows/master.yml

```diff
@@ -111,9 +114,12 @@ jobs:
         conda install -c conda-forge --yes pandas==$PANDAS_VERSION pyarrow==$PYARROW_VERSION
         conda install -c conda-forge --yes --freeze-installed --file requirements-dev.txt
         conda list
+    - name: Run tests
```

Comment:
Maybe this couldn't work since they refresh the shell for every step,

So we need `conda activate test-environment` again.
(but it was not working, that's why i integrated the steps)

Reply:
Yeah, I would like to resolve this issue in this PR. Let me take a look.

Reply:
yes I hope you solve it!